src/pages/LibrariesPage.vue

```diff
@@ -1,10 +1,44 @@
 <template>
-  <q-page class="flex q-pl-md bg-grey-1">
+  <q-page class="column q-pa-md bg-grey-1">
     <h4
-      class="q-ma-none q-mt-md"
+      class="q-ma-none q-mt-md q-mb-lg"
       data-cy="page_libraries_title"
     >
       {{ $t('LibrariesPage.text.title') }}
     </h4>
+    <q-table
```

Comment:
Put table in a LibrariesTable like the issues says with column setup and other

Reply:
done